tests/yam/agama/boot_agama.pm

```diff
@@ -22,6 +23,15 @@ BEGIN {
 use bootloader_s390;
 use bootloader_zkvm;
 
+sub expand_profile {
```

Comment:
this will be needed for other architecture so it cannot go at the test level.

Reply:
Could this be moved inside `package Yam::Agama::patch_agama_base` ?

Reply:
no, patch is for patching not for that, it has to be accessible for other test modules as a library, for example it could be added here for baremetal:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20777/files#diff-f8329f3474ec62c7e8b241c5be029319db687b91e770ec1f2327dc96ed9c0a97R165

Reply:
Please, use autoyast, I have concerns about creating yet another library to maintain just because we didn't have capacity rework the other booting mechanisms.

Reply:
Moved to `autoyast.pm`